ports/ecm/portfile.cmake

```
@@ -1,9 +1,11 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO KDE/extra-cmake-modules
-    REF 5c191fa1d2383c69ddc075395e04f84b7679613a # v5.74.0
-    SHA512 9a520e2cb45f2891cb5b463409063549904574e2ad0c4f77987555fc9aac68589695344a3acaddcb95d2739458031ece489ff9635c7d713c6309e4f9027f6ef7
+    REF v5.75.0
+    SHA512 486845a91967c89dec42f9120757c7b883006713d0957e9bd21dfacb2911225fe45680590ef3dc5e0d45a9dbc81cf99e1514fb4922b1a027bcba3864da883076
     HEAD_REF master
+    PATCHES
+      "do-not-override-custom-clang-format-files.patch"
```

```suggestion
    PATCHES
        "do-not-override-custom-clang-format-files.patch"
```
There should be 4 spaces.